Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -892,25 +893,32 @@ namespace AzToolsFramework
     {
         if (!m_prefabsAreEnabled)
         {
-            return m_isLevelEntityEditor ? InspectorLayout::LEVEL : InspectorLayout::ENTITY;
+            return m_isLevelEntityEditor ? InspectorLayout::OPENPREFABCONTAINERENTITY : InspectorLayout::ENTITY;
```

Comment:
nit: This might be outside the scope of this change but `UPPERCASE` names are reserved for macros, if this is an enum it should really be `UpperCamelCase`, but this can be deferred for another PR

Reply:
Agreed, it wasn't that invasive so I fixed it :D